addons/medical/functions/fnc_unconsciousPFH.sqf

```diff
@@ -135,6 +135,19 @@ if (!local _unit) exitwith {
     [(_this select 1)] call cba_fnc_removePerFrameHandler;
 };
 
+if (GVAR(maxUnconsciousTime) > 0 && {!(_unit getvariable [QGVAR(inReviveState), false])} && {(GVAR(level) == 1)}) then {
+    private ["_maxUnitUnconsciousTime"];
+    _maxUnitUnconsciousTime = if ((_unit getvariable [QGVAR(maxDynamicUnconsciousTime), 0]) > 0) then {
```

Comment:
It was a request from the person who asked me to do the adjustment - that it will not be one set value for all the time to the point where if you know that its five minutes long and epi is not available, then you know you have enough time to go to the toilet.